packages/@n8n/config/src/configs/database.ts

```diff
@@ -0,0 +1,159 @@
+import { Config, FromEnv, Nested } from '../decorators';
+
+@Config
+class LoggingConfig {
+	/** Typeorm logging enabled flag. @default false */
+	@FromEnv('DB_LOGGING_ENABLED')
+	readonly enabled: boolean = false;
```

Comment:
Ideally there should be a way to have the effect of `readonly` but without overly narrowing the type and without having to specify it... Not sure if possible but I'll look if there's a way.

Reply:
Not possible :/